Bottom: c6336f906bb32734e77977e84e7819f395cbfe54
Top:    67cf298c63fd5a648133a33eda2fa7dae45b27e4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 17:26:26 +0000

Refresh of polygon-c-refactor

---

diff --git a/src/polygon.c b/src/polygon.c
index 3e65cad..eabb6b5 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2011,9 +2011,8 @@ NoHolesPolygonDicer (PolygonType *p, const BoxType * clip,
 {
   POLYAREA *main_contour, *cur, *next;
 
-  main_contour = poly_Create ();
   /* copy the main poly only */
-  poly_Copy1 (main_contour, p->Clipped);
+  poly_Copy0 (&main_contour, p->Clipped);
   /* clip to the bounding box */
   if (clip)
     {
